content/articles/srv-record.markdown

```diff
@@ -0,0 +1,35 @@
+---
+title: What is a SRV Record?
+excerpt: Welcome to DNSimple. This page is about the SRV record. Hosted DNS has never been this easy.
+categories:
+- DNS
+---
+
+# What is a SRV Record?
+
+An SRV record defines a Service.
```

Comment:
"Service" is kinda generic and can be confused with our one-click "Services". The paragraph below this one is much better. Either be more specific about what type of service, or consider removing this sentence.